File: Algorithms/412. Fizz Buzz/golang/solution.string-concatenation.go
func fizzBuzz(n int) []string {
	ans := []string{}

	for i := 1; i <= n; i++ {
		numAnsStr := ""
		if i%3 == 0 {
			numAnsStr += "Fizz"
		}
		if i%5 == 0 {
			numAnsStr += "Buzz"
		}
		if numAnsStr == "" {
			numAnsStr = strconv.Itoa(i)
		}
		ans = append(ans, numAnsStr)
	}
	return ans
}

File: Algorithms/412. Fizz Buzz/javascript/solution.string-concatenation.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function (n) {
    const ans = [];

    for (let num = 1; num <= n; num++) {
        let numAnsStr = '';
        if (num % 3 === 0) {
            numAnsStr += 'Fizz';
        }
        if (num % 5 === 0) {
            numAnsStr += 'Buzz';
        }
        if (numAnsStr === '') {
            numAnsStr = num.toString();
        }
        ans.push(numAnsStr);
    }
    return ans;
};
